aviary/subsystems/propulsion/gearbox/model/gearbox_premission.py

```diff
@@ -44,18 +44,25 @@ def setup(self):
                                             'RPM_out'],
                            promotes_outputs=['torque_max'])
 
-        # Simple gearbox mass will always produce positive values for mass based on a fixed specific torque
-        self.add_subsystem('mass_comp',
-                           om.ExecComp('gearbox_mass = torque_max / specific_torque',
-                                       gearbox_mass={'val': 0.0, 'units': 'kg'},
-                                       torque_max={'val': 0.0, 'units': 'N*m'},
-                                       specific_torque={'val': 0.0, 'units': 'N*m/kg'},
-                                       has_diag_partials=True),
-                           promotes_inputs=['torque_max',
-                                            ('specific_torque', Aircraft.Engine.Gearbox.SPECIFIC_TORQUE)],
-                           promotes_outputs=[('gearbox_mass', Aircraft.Engine.Gearbox.MASS)])
+        if self.options["simple_mass"]:
+            # Simple gearbox mass will always produce positive values for mass based on a fixed specific torque
+            self.add_subsystem(
+                'mass_comp',
```

Comment:
If include_gearbox = false probably want to set this mass to zero if you have not done that elsewhere already.